tests/e2e/specs/dashboard/WPSidebarNav.js

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Unconventional file name for our repo :)

Maybe `adminMenu.js`? Or just move these tests to the existing `tests/e2e/specs/wordpress/adminMenu.js` file

Reply:
lol yea felt weird as I was writing it. thanks